<!--
Google IO 2012/2013 HTML5 Slide Template

Authors: Eric Bidelman <ebidel@gmail.com>
         Luke Mahé <lukem@google.com>

URL: https://code.google.com/p/io-2012-slides
-->

<!DOCTYPE html>
<html>
<head>
  <title></title>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="chrome=1">
  <!--<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">-->
  <!--<meta name="viewport" content="width=device-width, initial-scale=1.0">-->
  <!--This one seems to work all the time, but really small on ipad-->
  <!--<meta name="viewport" content="initial-scale=0.4">-->
  <meta name="apple-mobile-web-app-capable" content="yes">
  <link rel="stylesheet" media="all" href="theme/css/default.css">
  <link rel="stylesheet" media="only screen and (max-device-width: 480px)" href="theme/css/phone.css">
  <base target="_blank"> <!-- This amazingness opens all links in a new tab. -->
  <script data-main="js/slides" src="js/require-1.0.8.min.js"></script>
</head>
<body style="opacity: 0">

<slides class="layout-widescreen">

<!-- Slide 1 -->

  <slide class="title-slide segue nobackground">
    <!--<aside class="gdbar"><img src="images/google_developers_icon_128.png"></aside> -->
    <!-- The content of this hgroup is replaced programmatically through the slide_config.json. -->
    <hgroup class="auto-fadein">
      <h1 data-config-title><!-- populated from slide_config.json --></h1>
      <h2 data-config-subtitle><!-- populated from slide_config.json --></h2>
      <p data-config-presenter><!-- populated from slide_config.json --></p>
    </hgroup>
  </slide>

<!-- Slide 2 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
          <li>Give credit where credit is due</li>
          <li>My motivation for preparing slides in this manner</li>
          <li>For this presentation, I didn't use animated lists to make it easier to transition through slides</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>About this presentation</h2>
      <h3>Technical aspects</h3>
    </hgroup>

    <article>
      <ul>
        <li>This presentation was written in HTML using the framework (io-2012-slides) developed by two Google developers (Eric Bidelman and Luke Mahé)</li>
        <li>It is version controlled on <a href="https://github.com/davetang/tki/tree/gh-pages">GitHub</a> and can be openly viewed on the web using GitHub pages</li>
        <li>Pressing 'p' toggles speaker notes for the current slide</li>
        <li>Pressing 'f' toggles fullscreen viewing</li>
        <li>Pressing 'w' toggles widescreen</li>
        <li>Pressing 'o' toggles overview mode</li>
        <li>Pressing 'ESC' toggles off these goodies</li>
      </ul>
    </article>

  </slide>

<!-- Slide 3 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
          <li>Just a short description and not my life story.</li>
          <li>I've been hired as a research assistant at CSIRO and UQ.</li>
          <li>Some of my work can be found at:</li>
          <li><a href="http://www.biomedcentral.com/qc/1471-2164/10/188">Cow tRNAs</a></li>
          <li><a href="http://nar.oxfordjournals.org/content/41/3/e44">Barcode bias</a></li>
          <li><a href="https://github.com/davetang/fantom5_repeat">Expression of repetitive elements</a> (currently private)</li>
          <li>I'm not presenting my work on the blood transcriptome</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>About this presentation</h2>
      <h3>Outline</h3>
    </hgroup>

    <article>
      <ul>
        <li>Introduction of myself</li>
        <li>Description of my support positions</li>
        <ul>
           <li>I will only talk about these briefly</li>
           <li>At The Commonwealth Scientific and Industrial Research Organisation (CSIRO)</li>
           <li>At The University of Queensland (UQ)</li>
        </ul>
        <li>Description of my research</li>
        <ul>
           <li>On the annotation of transfer RNAs as part of the Bovine Genome project</li>
           <li>On detecting biases in transcriptome sequencing</li>
           <li>On the analysis of repetitive elements using transcriptome sequencing</li>
        </ul>
      </ul>
    </article>

  </slide>

<!-- Slide 4 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
          <li>Everyone always asks me how I ended up in PNG. I didn't have a choice as a newborn.</li>
          <li>Leaving Brisbane to go to Japan was one of the hardest decisions I had to make (but I'm extremely glad I decided to move to Japan)</li>
          <li>Technically British National Overseas</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>About me</h2>
    </hgroup>

    <article>
      <ul>
        <li>I was born in Hong Kong but left after my first birthday</li>
        <li>I spent the next 15 years of my life growing up in Papua New Guinea</li>
        <li>I then went to Brisbane for high school and university</li>
        <li>Initially I wanted to pursue a PhD after my Honours but decided to get more experience</li>
        <li>I worked as a bioinformatics research assistant at CSIRO and UQ</li>
        <li>I decided I was ready to pursue a PhD in 2010 and left for Japan</li>
        <li>I have both British and Australian citizenship</li>
      </ul>
    </article>

  </slide>

<!-- Slide 5 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>I initially joined CSIRO as a work experience trainee without salary</li>
           <li>I helped select candidate SNPs for the <a href="http://www.sheephapmap.org/pilot.php">1.5k Pilot SNP Array</a></li>
           <li><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004668">A Genome Wide Survey of SNP Variation Reveals the Genetic Structure of Sheep Breeds</a>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>CSIRO</h2>
    </hgroup>

    <article>
      <ul>
         <li>I helped maintain the local instance of GBrowse (a genome browser written in Perl) that was used by the division</li>
            <ul>
               <li>This included preparing new genome browser tracks, maintaining the MySQL database, and the Apache2 web server</li>
            </ul>
         <li>I developed a GBrowse plugin that allowed a user to download DNA sequence from the browser</li>
         <li>Added additional functionality to the browser using Perl and JavaScript</li>
         <li>I assisted with the development of the first sheep SNP array by selecting the candidate SNPs</li>
      </ul>
    </article>

  </slide>

<!-- Slide 6 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>In addition to building tracks, I also implemented a plugin to view BAC sequences from the genome browser, which also displayed various sequence annotations</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>GBrowse</h2>
    </hgroup>
    <article class="flexbox vcenter">
      <img src="images/Figure_S4.png" alt="GBrowse" title="A Genome Browser written in Perl">
      <footer class="source"><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004668#s5">http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004668#s5</a></footer>
    </article>
  </slide>

<!-- Slide 7 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>I was also one of the system administrator of the lab and performed other assorted tasks for lab members, significant enough to be included as an author.</li>
           <li>I also briefly assisted in setting up infrastructure for the lab as they started in the International Cancer Genome Consortium Cancer Genome Project.</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>UQ</h2>
    </hgroup>

    <article>
      <ul>
         <li>I joined UQ as a Laboratory Information Management System developer for <a href="http://www.gudmap.org/">GUDMAP</a></li>
         <li>I developed and implemented a <a href="http://uqgudmap.imb.uq.edu.au/tools">suite of tools</a> to assist the project</li>
         <li>I developed a <a href="http://uqgudmap.imb.uq.edu.au/riboprobe_design/">tool</a> to design riboprobes for validating microarray results</li>
         <li>I maintained all the data generated by the project</li>
      </ul>
    </article>

  </slide>

<!-- Slide 8 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>I did my Honours work in a molecular evolution lab and applied the knowledge I learned there to this project</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>Bovine tRNAs</h2>
    </hgroup>

    <article>
      <ul>
         <li>I was in charge of annotating putatively functional bovine tRNAs as part of the Bovine Genome Consortium</li>
         <li>The bovine genome contains a large amount of tRNA-derived SINEs, which would come up as false positives</li>
         <li>RepeatMasker was used to remove the SINEs</li>
         <li>Comparative genomics was used to select a set of putatively functional tRNA genes</li>
         <li>As well as creating a set of high-confidence tRNA genes, this work suggested the evolution of a specific bovine <a href="http://www.ncbi.nlm.nih.gov/pubmed/9497276">tRNA via a point mutation</a></li>
      </ul>
    </article>

  </slide>

<!-- Slide 9 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>The filtered set of bovine tRNAs had a similar distribution to tRNAs in other genomes</li>
           <li>Dengrogram of bovine tRNAs show that most tRNAs for the same amino acid are phylogenetically closer to each other, but not always</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>Bovine tRNAs</h2>
    </hgroup>
    <article class="flexbox vcenter">
      <img src="images/1471-2164-10-188-2.jpg" alt="dist_trna" title="Distribution of tRNAs in various genomes">
      <img src="images/1471-2164-10-188-3.jpg" alt="dendrogram" title="Dendrogram of bovine tRNAs">
      <footer class="source"><a href="http://www.biomedcentral.com/qc/1471-2164/10/188">Analysis of the complement and molecular evolution of tRNA genes in cow</a></footer>
    </article>
  </slide>

<!-- Slide 10 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>When I was first analysing nanoCAGE data, nothing made any sense! The technology was inherently noisier than CAGE but even then the variance was huge across technical replicates. Finally I realised that there was a barcode bias</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>My introduction to transcriptome sequencing</h2>
    </hgroup>

    <article>
      <ul>
         <li>Prior to my PhD, I had no experience in RNA sequencing (RNA-Seq), despite coming from a lab that was one of the first to perform RNA-Seq</li>
         <li>Initially I worked with a variant of Cap Analysis Gene Expression (CAGE) called nanoCAGE, which is based on a molecular mechanism known as template switching</li>
         <li>The scientist who made the nanoCAGE libraries, by mistake, used different barcodes for technical replicates</li>
         <li>After a while, I realised that the libraries were clustering by the barcode sequence used for a particular library</li>
         <li>Thus I discovered barcode bias and implemented a computational strategy to mitigate this bias and helped improve the nanoCAGE technology</li>
      </ul>
    </article>

  </slide>

<!-- Slide 11 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>Strand invasion takes place when the cDNA product has sequence complementary to the template switching (TS) oligo</li>
           <li>Strand invasion leads to a false TSS and can be easily detected</li>
           <li>I devised a simple computational strategy that removed strand invasion products and helped improve the nanoCAGE protocol</li>
           <li>The scatter plots show transcripts that are differentially expressed (in red) among technical replicates in A and B but not in C, when the filtering was performed</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>Strand invasion and mitigating the bias</h2>
    </hgroup>
   <article class="flexbox vcenter">
      <img src="images/F1.large.jpg" alt="strand_invasion" title="Schematic of strand invasion">
      <img src="images/F3.large.jpg" alt="de" title="Differential expression after removing barcode bias">
      <footer class="source">See original <a href="http://nar.oxfordjournals.org/content/41/3/e44/F1.large.jpg">figure 1</a> and <a href="http://nar.oxfordjournals.org/content/41/3/e44/F3.large.jpg">figure 3</a></footer>
    </article>
  </slide>

<!-- Slide 12 -->

  <slide>

    <!-- Speaker notes -->
    <aside class="note">
      <section>
        <ul>
           <li>This has been one of the most technically challenging projects I've worked on</li>
           <li>The main finding is the specificity of expression coming from repetitive elements</li> 
           <li>This work is almost ready for submission</li>
        </ul>
      </section>
    </aside>

    <hgroup>
      <h2>Analysing expression from repetitive elements</h2>
    </hgroup>

   <article class="flexbox vcenter">
      <img src="images/figure_1.png" alt="de" title="Differential expression after removing barcode bias">
    </article>

  </slide>

<!-- Last slide -->

  <slide class="thank-you-slide segue nobackground">
    <article class="flexbox vleft auto-fadein">
      <h2>Thank you for your attention!</h2>
    </article>
    <p class="auto-fadein" data-config-contact>
      <!-- populated from slide_config.json -->
    </p>
  </slide>

  <slide class="backdrop"></slide>

</slides>

<script>
var _gaq = _gaq || [];
_gaq.push(['_setAccount', 'UA-XXXXXXXX-1']);
_gaq.push(['_trackPageview']);

(function() {
  var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
  ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
  var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
})();
</script>

<!--[if IE]>
  <script src="http://ajax.googleapis.com/ajax/libs/chrome-frame/1/CFInstall.min.js"></script>
  <script>CFInstall.check({mode: 'overlay'});</script>
<![endif]-->
</body>
</html>
